### DISCUSSION TITLE
Run Github Actions when issue contains a specific label

### DISCUSSION THREAD

**Discussion by qianxi0410** [community/community#27722] [Actions] [answered]
The workflow is only triggered when an issue with a specific tag is changed (modified, created, deleted, closed), is there a way to do this.

**Comment by airtower-luna** [comment, marked as answer]
Not in the workflow trigger event itself.

At the job level you can [attach an `if:` condition](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif) that checks `github.event.issue.labels` for whatever labels you want, and skips the job otherwise. There's actually [an example in the docs](https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-array) that does just that:

```yaml
if: contains(github.event.issue.labels.*.name, 'bug')
```

**Reply by Samarapdilha** [reply]
Oi
